### PR TITLE
Update workflows events to ignore tests PRs on pull requests

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update workflows events to ignore tests PRs on pull requests
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | no
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | N/A

This was a suggestion for @kpodemski to not run some workflows when only tests are updated
Docs : 
- [Paths-ignore](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore)

### CODE REVIEW

**Review by kpodemski — COMMENTED** [review]
Love it! I think this should speed up the tests drastically for UI tests. Later we could think about deciding on the correlation between tests and paths for other things too :)

**Review by kpodemski — DISMISSED** [review]

File: .github/workflows/behaviour.yml
Comment (jolelievre) [inline .github/workflows/behaviour.yml]:
```suggestion
      - '**.x'
```
With a wildcard we would anticipate the future branches as well no? Like 8.1.x, 9.0.x, ...

Comment (jolelievre) [reply .github/workflows/behaviour.yml]: Other alternative
```suggestion
      - '*.*.x'
      - '1.7.8.x'
```

Comment (boubkerbribri) [reply .github/workflows/behaviour.yml]: First suggestion is better to take out 1.7.8.x too

**Review by jolelievre — DISMISSED** [review]
Thanks @boubkerbribri

File: .github/workflows/behaviour.yml
Comment (Progi1984) [inline .github/workflows/behaviour.yml:11]: Ok for `Do not run when only tests are updated`

But I disagree `Do not run after merging a pull request`

Because a PR can break the CI after a merge (for example, if the PR is old and has no conflicting files).

**Review by Progi1984 — COMMENTED** [review]
Ok for Do not run when only tests are updated

But I disagree Do not run after merging a pull request

Because a PR can break the CI after a merge (for example, if the PR is old and has no conflicting files).

**Comment by boubkerbribri** [comment]
> Ok for Do not run when only tests are updated
> 
> But I disagree Do not run after merging a pull request
> 
> Because a PR can break the CI after a merge (for example, if the PR is old and has no conflicting files).

I agree, but is it used ? I don't that there is a check on workflows after merge (but I'm maybe mistaken)

**Review by lartist — DISMISSED** [review]

**Comment by Progi1984** [comment]
@boubkerbribri FYI the merger receive an email if the CI has a failure after merge. And that helps (a lot) to know where the CI has been broken when we detect a fail. It is like a integration check CI.

**Comment by boubkerbribri** [comment]
> @boubkerbribri FYI the merger receive an email if the CI has a failure after merge. And that helps (a lot) to know where the CI has been broken when we detect a fail. It is like a integration check CI.

I'm ok to keep it. @PrestaShop/prestashop-maintainers WDYT ?

**Comment by kpodemski** [comment]
>FYI the merger receive an email if the CI has a failure after merge.

I didn't know this, I think I've never received such an email, but if that's ok, then we should keep it.

ping @PrestaShop/prestashop-maintainers again :)

**Review by jolelievre — APPROVED** [review]
Thanks @boubkerbribri

**Comment by jolelievre** [comment]
Thanks @boubkerbribri 

Do we need to update these workflow files for 8.0.x as well? Or the files from the default branch are used as references for all the branches?

**Comment by boubkerbribri** [comment]
> Thanks @boubkerbribri
> 
> Do we need to update these workflow files for 8.0.x as well? Or the files from the default branch are used as references for all the branches?

Good question, I don't think so, we don't create much PR targeting 8.0.x

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
thanks @boubkerbribri 🎉

**Pull request merged**
